x-pack/plugins/security_solution/server/endpoint/routes/actions/list_handler.ts

```diff
@@ -61,16 +59,6 @@ export const actionListHandler = (
     const esClient = (await context.core).elasticsearch.client.asInternalUser;
 
     try {
-      const indexExists = await doesLogsEndpointActionsIndexExist({
```

Comment:
Hey! As I can remember, this is used in order to know if the index is empty or if it does not exists yet. 

When we do a search in the API, if the response is 404 with a `index_not_found_exception` body, it means the index does not exists and allow us to display an empty message in the frontend side. If the search has filters that returns no results, then it will return an empty array (as done before).

This is the pr: https://github.com/elastic/kibana/pull/140942

Same thing could be done doing an extra call (as we do for our artifact list pages when the search does not return any result). But we decided to go with this approach as this is something will only happen when there is no action yet on the response.

@ashokaditya can give us more details about this